cc/toolchains/impl/toolchain_config_info.bzl

```diff
@@ -63,7 +63,7 @@ def _get_known_features(features, capability_features, fail):
     for ft in capability_features + features:
         if ft.name in feature_names:
             other = feature_names[ft.name]
-            if other.overrides != ft and ft.overrides != other:
+            if other.overrides != ft and ft.overrides != other and ft.label != other.label:
```

Comment:
When multiple tools have the same capability, they end up in the list multiple times

Reply:
Thanks, I'll rebase and remove this change!